### ISSUE TITLE
Stop using the last argument as keyword parameters in serverless.rb

### ISSUE THREAD

**Issue by pwrmiller** [stelligent/cfn-model#82]
Running cfn_nag_scan 0.6.0 with cfn_model 0.5.0, scanning against a template using the Serverless transform, we see warnings related to deprecated Ruby functionality:

```
# cfn_nag_scan --input-path=./.aws-sam/build/template.yaml
/usr/local/Cellar/gem-cfn-nag/0.6.0/gems/cfn-model-0.5.0/lib/cfn-model/transforms/serverless.rb:115: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/local/Cellar/gem-cfn-nag/0.6.0/gems/cfn-model-0.5.0/lib/cfn-model/transforms/serverless.rb:206: warning: The called method `lambda_function' is defined here
```

**Comment by erickascic** [comment]
@pwrmiller sorry for the delay.  which version of ruby are you running on - is that 2.7?

**Comment by erickascic** [comment]
fixed in 0.5.1.  pushing to cfn_nag
